ARExperiment/ARExperiment/CVPixelBuffer+Helpers.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
```

Comment:
Do we have the rights for this code? Should you import the information of the author and the license?

Reply:
I added the license to all the files we did not write and set them on its own folder